bigchaindb/models.py

```diff
@@ -118,8 +118,36 @@ def validate(self, bigchain):
 
 
 class Block(object):
+    """A Block bundles up to 1000 Transactions. Nodes vote on its validity.
+
+        Attributes:
```

Comment:
Just noticed this now, but all the docstrings in models.py are consistently indented oddly. Normally don't they line up with the first `"`?